spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/data/SparkParquetWriters.java

```diff
@@ -299,6 +301,28 @@ public void write(int repetitionLevel, byte[] bytes) {
     }
   }
 
+  private static PrimitiveWriter<UTF8String> uuids(ColumnDescriptor desc) {
```

Comment:
is this required for this change?  (Couldn't figure out where it's used).  Maybe we can separate it out for people more familiar to take a look at this part?